common-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/common/client/LoginPage.java

```diff
@@ -478,10 +489,16 @@ public void onClick(ClickEvent event) {
         // if not, the login input text is set to the cacheLogin
         if (getCookieUserName() == null) {
             if (cacheLogin != null) {
-                form.setValue("login", cacheLogin);
+                String username = cacheLogin.split("\\\\")[1];
```

Comment:
the split works only when there is a domain present in the cacheLogin, it will throw an ArrayIndexOutOfBoundException if not.

Reply:
also the cached domain could be used to set the value of the domainIItem field